src/Hangman/Domain/Repositories/GamesRepository.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { GameDto } from '../AggregateRoot/GameDto';
+import { Game } from '../AggregateRoot/Game.aggregate';
+
+@Injectable()
+export class GamesRepository {
+  async startGame(gameDto: GameDto) {
+    const game = new Game(undefined);
```

Comment:
Dit is meer een implementatie detail en ik zou niet aanpassen maar je zou je kunnen afvragen: Zou je ooit een instantie mogen aanmaken die geen gameId heeft? in dat geval zou je hier je command moeten meegeven.